models.py

```diff
@@ -209,7 +209,7 @@ class Video(Medium):
 
 @slugify.apply
 class Audio(Medium):
-    pass
+    href = StringField(required=True)
```

Comment:
Ha. This class was written exactly 1 year ago, tomorrow. It's about time we got around to filling it in!